phpunit.xml.dist

```diff
@@ -13,7 +13,7 @@
 
         <testsuite name="phpcr">
             <directory>./Tests/Functional</directory>
-            <exclude>./Tests/Functional/Doctrine/Orm</exclude>
+            <exclude>./Tests/Functional/Doctrine/Phpcr</exclude>
```

Comment:
nope. it excluded the orm tests. now you try to exclude the phpcr tests and run the orm tests.